frontend/lib/src/components/widgets/AudioInput/AudioInput.tsx

```diff
@@ -203,16 +203,24 @@ const AudioInput: React.FC<Props> = ({
   )
 
   const handleClear = useCallback(
-    ({ updateWidgetManager }: { updateWidgetManager?: boolean }) => {
+    ({
+      updateWidgetManager,
+      deleteFile,
+    }: {
+      updateWidgetManager?: boolean
+      deleteFile?: boolean
```

Comment:
**nit:** Why make this optional if every callsite explicitly provides it anyway? I'm personally a fan of being explicit so that the type checker helps us catch mistakes proactively, but I also understand we don't have any set paradigm on this in the codebase, hence why this is just a nit.

Reply:
it was optional because i was only going to specify it in some locations and then i was like ehhh might as well specify it everywhere. let me get that `?` out of there